python/tank_vendor/shotgun_deploy/descriptor_bundle.py

```diff
@@ -41,8 +41,7 @@ def get_version_constraints(self):
 
         manifest = self._io_descriptor.get_manifest()
 
-        if manifest.get("requires_shotgun_version") is not None:
-            constraints["min_sg"] = manifest.get("requires_shotgun_version")
+        constraints["min_sg"] = manifest.get("requires_shotgun_version", "5.0.0")
```

Comment:
Please make `5.0.0` a constant and add a comment explaining where the value comes from.